packages/react-popup/Spec.md

```diff
@@ -34,30 +388,115 @@ _List the **Props** and **Slots** proposed for the component. Ideally this would
 
 _Describe what will need to be done to upgrade from the existing implementations:_
 
-- _Migration from v8_
-- _Migration from v0_
+### v8
+
+- `onDismiss` should listen to `onOpenChange`.
+- `preventDismissOnEvent` no longer exists, `onOpenChange` will return the associated event, so this functionality is still possible.
+- Removed `onRestoreFocus`. `Popup` will focus the trigger when closed by default. Any other behaviour can be done through an effect by users.
+- Removed `onPositioned`.
+- Removed `doNotLayer` with `inline`.
+- Removed all styling props supported by `Callout`. `makeStyles` should be used instead.
+- Removed `coverTarget`, use `offset` callback instead.
+- Removed `setInitialFocus`. If `trapFocus` is used, first focusable element is used. Users must handle other specific cases.
+
+### v0
+
+> TODO Discuss: v0 autoFocus -> only used in app-profile-card
+
+- No more `autoFocus` for scenarios without `trapFocus`. Users should handle this scenario manually.
+- No more `tabbableTrigger`. Users can do this with their own trigger element.
+- No `PopoverContent` props, v0 duplicated props from `Popover` to `PopoverContent`, all props should be declared on converged `Popover`.
```

Comment:
nit: Might be good to move this into a separate `MIGRATION.md` file to follow the pattern in other packages.

Reply:
yup will do that in a future PR